tests/connection/connection.cfg

```diff
@@ -2,3 +2,28 @@
 hostname=192.168.1.1
 username=admin
 password=password
+
+[remote]
```

Comment:
`remote` is too generic a term. we should probably specify what does this refer to `remote-backup` or `remote-techsupport` or ?

similarly for the others.. even if this means duplicating some config, we should do it.